### PR TITLE
Replace "weight" with "parameter".rs

### PR DESCRIPTION
**Pull request by Expertium**

### CODE REVIEW

File: src/optimal_retention.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ use ndarray::{s, Array1, Array2, Ix0, Ix1, SliceInfoElem, Zip};
 use ndarray_rand::rand_distr::Distribution;
 use ndarray_rand::RandomExt;
 use rand::{
-    distributions::{Uniform, WeightedIndex},
+    distributions::{Uniform, parameteredIndex},
```
Comment (L-M-Sherlock) [inline src/optimal_retention.rs]: It's not the weight of FSRS. It's imported from other package. Please revert this change.

**Review by L-M-Sherlock — APPROVED** [review]
LGTM

File: src/optimal_retention.rs (hunk 1 above)
Comment (Expertium) [reply src/optimal_retention.rs]: My bad, I used auto find and replace
